interfaces/builtin/steam_support.go

```diff
@@ -201,7 +199,7 @@ umount /,
 /usr/** ixr,
 deny /usr/bin/{chfn,chsh,gpasswd,mount,newgrp,passwd,su,sudo,umount} x,
 /run/host/** mr,
-/run/pressure-vessel/** mrw,
+/*/pressure-vessel/** mrw,
```

Comment:
Yeah, I purposely didn't widen this any further for that reason. We could widen it if it's ok security-wise

Reply:
I think probably the restriction you ideally want here is something like "after doing the `pivot_root` into its new root directory, pressure-vessel can write wherever it wants to write" - but I suspect that isn't actually representable, because AppArmor only sees paths relative to the current root directory, and doesn't know when we did the `pivot_root`.

Assuming that pressure-vessel will use a directory named `pressure-vessel` (somewhere) for its internal state doesn't seem too bad in terms of probability of regressing. For a long time we were using `/run/pressure-vessel`, but we had to stop using that path for the `ld.so.cache`, for better compatibility with an x86-on-ARM CPU emulation framework (FEX-Emu) that fakes a chroot/overlayfs in user-space.